### ISSUE TITLE
uploaded predictor visibility not changeable?

### ISSUE THREAD

**Issue by yibeichan** [neuroscout/neuroscout#1099]
**Describe the bug**
In the [document](https://neuroscout.github.io/neuroscout-docs/web/upload.html), the uploaded predictor visibility is `public`.
But there is no option regarding setting this visibility when uploading predictors. And, by default, I got `private`.

**To Reproduce**
regular uploading steps.

**Expected behavior**
1. Should this attribute be changeable or not? Is it by default private?
2. Is it possible to make uploaded predictors editable (or deletable)? (after it's being uploaded

**Screenshots**
In the document
![image](https://user-images.githubusercontent.com/20402800/210194886-693694a0-0d1d-4a5d-a7e1-0106a0a277d0.png)
My result
![image](https://user-images.githubusercontent.com/20402800/210194864-5ff938c3-3c9c-4e40-9797-496b1043ffd5.png)

**Desktop (please complete the following information):**
 - Browser: Firefox (MacOS)

Thanks!

**Comment by adelavega** [comment]
Hi Yibei,

It's actually the case that by default they are private-- I should make that more clear in the docs.

However, I can make yours public if you wish. The reason I did not make it changeable is I don't want everyone's uploads showing up for all other users, especially if there's no verification to their accuracy. 

However, I think at the minimum I should make it easier to make it public, even if by default the collections are still "private". If in the future they are causing clutter, we can re-think this, but I doubt this will be an issue for now.

After I verify that your uploads look good, I will make them public

**Comment by adelavega** [comment]
Editabilty is probably a bit tougher, but deletable makes sense to me (although if they are already used in an analysis--- maybe not? Maybe we can do a soft delete, i.e. make them hidden/private again?)

**Comment by yibeichan** [comment]
Makes sense! Let me know if my uploads have any errors (e.g., formatting). If so, I'll fix them.
